frontend/src/Context/SessionProvider/session.tsx

```diff
@@ -231,130 +209,23 @@ const SessionProvider = ({ children }: SessionProviderProps): ReactElement => {
   }, [moveSubscriberToTopOfDisplayOrder, setActiveSpeakerIdAndRef]);
 
   const { user } = useUserContext();
-  const [changedStream, setChangedStream] = useState<ChangedStreamType | null>(null);
-  const [connections, setConnections] = useState<Connection[]>([]);
   const [connected, setConnected] = useState(false);
 
   /**
    * Handles changes to stream properties. This triggers a re-render when a stream property changes
-   * @param {ChangedStreamType} params - Object containing details about the changed stream.
-   */
-  const handleStreamPropertyChanged = ({
-    stream,
-    changedProperty,
-    newValue,
-    oldValue,
-  }: ChangedStreamType) => {
-    setChangedStream({ stream, changedProperty, newValue, oldValue, token: uuid() });
-  };
-
-  /**
-   * Handles the creation of a new connection. Adding it to the connection array
-   * @param {ConnectionEventType} e - The connection event object.
-   */
-  const handleConnectionCreated = (e: ConnectionEventType) => {
-    setConnections((prevConnections) => [...prevConnections, e.connection]);
-  };
-
-  /**
-   * Handles the destruction of a connection. Removing it from the connections array
-   * @param {ConnectionEventType} e - The connection event object.
    */
-  const handleConnectionDestroyed = (e: ConnectionEventType) => {
-    setConnections((prevConnections) =>
-      [...prevConnections].filter(
-        (connection) => connection.connectionId !== e.connection.connectionId
-      )
-    );
```

Comment:
These two weren't used at all. It may have been leftovers from our call data function from the beginning.